Gulpfile.js

```diff
@@ -55,10 +55,10 @@ if(debugMode) {
   abstract out the process of making a bundler, and then leave it open to manual triggering
 */
 var indexBundler = bundlerBuilder('./src/js/index.jsx', 'index.js', true);
```

Comment:
Pretty sure we only need the one bundle now.  If anyone else knows any differently I'll roll back the changes made to the gulpfile.

Reply:
So, at the moment we only need one bundler, as all the components can be accessed in one place.  
I think the way the routing is structured at the moment  does a good job, but I am open to suggestions if we think we might want to create different bundles

Reply:
Let's keep with the one for now considering everything is in the single application now anyway